### PR TITLE
[en] Update go lang link common-parameters.md

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

**Review by PranitRout07 — COMMENTED** [review]
These changes are good . What are your thoughts changing these text font style to bold or italic , these changes will increase readability  of the page . 
Screenshot 2023-12-23 195045

**Comment by Arhell** [comment]
> These changes are good . What are your thoughts changing these text font style to bold or italic , these changes will increase readability of the page . Screenshot 2023-12-23 195045

No, this PR is only for replacing the link, if you want, take up your proposal.

**Comment by PranitRout07** [comment]
> > These changes are good . What are your thoughts changing these text font style to bold or italic , these changes will increase readability of the page . Screenshot 2023-12-23 195045
> 
> No, this PR is only for replacing the link, if you want, take up your proposal.

ok @Arhell .

File: content/en/docs/reference/kubernetes-api/common-parameters/common-parameters.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ When present, indicates that modifications should not be persisted. An invalid o
 
 ## fieldManager {#fieldManager}
 
-fieldManager is a name associated with the actor or entity that is making these changes. The value must be less than or 128 characters long, and only contain printable characters, as defined by https://golang.org/pkg/unicode/#IsPrint.
+fieldManager is a name associated with the actor or entity that is making these changes. The value must be less than or 128 characters long, and only contain printable characters, as defined by https://pkg.go.dev/unicode#IsPrint.
```
Comment (Ritikaa96) [inline content/en/docs/reference/kubernetes-api/common-parameters/common-parameters.md:63]:
```suggestion
fieldManager is a name associated with the actor or entity making these changes.
The value must be less than or 128 characters long and only contain printable
characters, as defined by https://pkg.go.dev/unicode#IsPrint.
```

Comment (Ritikaa96) [reply content/en/docs/reference/kubernetes-api/common-parameters/common-parameters.md:63]: small nit . please wrap the lines.

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]

**Review by Arhell — COMMENTED** [review]
/assign @sftim

**Comment by sftim** [comment]
/unassign
I'm happy for any approver to take the next step.

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Pull request merged**